src/main/java/art/heredium/controller/admin/AdminMembershipController.java

```diff
@@ -24,4 +30,11 @@ public ResponseEntity updateIsEnabled(
     this.membershipService.updateIsEnabled(membershipId, request.getIsEnabled());
     return ResponseEntity.ok().build();
   }
+
+  @GetMapping("/active")
```

Comment:
```suggestion
  @GetMapping("/users/active")
```
Để đỡ bị lẫn với Gói membership đang active

Reply:
Tôi đã thêm: 0c343c616ebb317a49bcfc6fbb06f939ed5e9bbe